### PR TITLE
[Community] Member Profile: Shivam Sood 

### LINKED ISSUE #3783: [Community] Member Profile:  Shivam Sood
**Current Behavior**

@Shivam-AfA  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @Shivam-AfA  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @Shivam-AfA 
- Twitter: @SoodX21
- LinkedIn: https://www.linkedin.com/in/shivam-sood-66005a1a0/
- Link to profile picture: https://drive.google.com/file/d/1D6e2DekUbQczKWob1YMTBNEdr5LsHXoS/view

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by UtkarshMishra12**
**Description**

This PR fixes #3783 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c05da929a08ca96238e8bb0f6aac61527a2a8fe8 at: https://63ff9c996c73af194944f6c4--layer5.netlify.app

**Review by Shivam-AfA — APPROVED** [review]
Thanks @UtkarshMishra12

File: src/collections/members/shivam-sood/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+name: Shivam Sood
+position: Contributor
+image_path: ./shivam-sood.jpg
+github: Shivam-AfA
+twitter: SoodX21
+linkedin: shivam-sood-66005a1a0
+location: Bathinda, Punjab, India
+bio: "Shivam is an aspiring software engineer, currently graduating from National Institute of Technology, Jalandhar. He aims to impact the world of engineering through never-ending passion and curiosity in advancing technologies. He is exploring open source with an inclination towards full-stack development, cloud-native technologies and DevOps in general."
+status: Active
+published: true
```
Comment (leecalcote) [inline src/collections/members/shivam-sood/index.mdx:11]: badges: meshmap

Comment (Chadha93) [inline src/collections/members/shivam-sood/index.mdx:11]:
```suggestion
published: true
badges: meshmap
```

Comment (Chadha93) [reply src/collections/members/shivam-sood/index.mdx:11]: Adding now.

**Comment by Chadha93** [comment]
@UtkarshMishra12, Good work here, thank you.

File: src/collections/members/shivam-sood/index.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+badges: meshmap
```
Comment (Chadha93) [inline src/collections/members/shivam-sood/index.mdx:12]:
```suggestion
badges: 
  - meshmap
```

**Comment by l5io** [comment]
🚀 Preview for commit 197b6bc22207a1ab31e85403d93f93deffab9b98 at: https://64011805a8023c2f85c96567--layer5.netlify.app

**Pull request merged**